base_location_nuts/wizard/nuts_import_view.xml

```diff
@@ -36,7 +35,7 @@
 <menuitem action="nuts_import_action"
           id="nuts_import_menu"
           name="Import NUTS 2013"
-          parent="base.menu_localisation"
+          parent="sales_team.menu_localisation"
```

Comment:
It's sad to have to add the `sales_team` dependency just for putting the menu somewhere. Do you think there's another place where we can put it and skip this big dependency just for that? 🤔 Or maybe we really need this for other reason?